internal/pkg/ccloudv2/flink_gateway.go

```diff
@@ -79,3 +80,8 @@ func (c *FlinkGatewayClient) GetStatementResults(orgId, environmentId, statement
 	res, r, err := req.Execute()
 	return res, errors.CatchCCloudV2Error(err, r)
 }
+
+func (c *FlinkGatewayClient) GetExceptions(orgId, environmentId, statementId string) (flinkgatewayv1alpha1.SqlV1alpha1StatementExceptionList, error) {
```

Comment:
Nit: We like to pass arguments into the function in the same order as they're used.
```suggestion
func (c *FlinkGatewayClient) GetExceptions(environmentId, statementId, orgId string) (flinkgatewayv1alpha1.SqlV1alpha1StatementExceptionList, error) {
```

Reply:
Reordered this and all other methods in GW client!